Packs/Core/ReleaseNotes/3_2_41.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### XQL Query Engine
+- Updated the Docker image to: *demisto/python3:3.12.8.3296088*.
+
+- Updated the XQL Query Engine integration to send metadata on playbook task and command to xql/start_xql_query endpoint.
```

Comment:
Updated the XQL Query Engine integration to send metadata on a playbook task and command to the xql/start_xql_query endpoint.